lib/main.dart

```diff
@@ -41,10 +42,12 @@ class Home extends StatefulWidget {
 class _HomeState extends State<Home> {
   final Network _network = Network();
   final _preGame = Game(turnOrder: PlayerColor.values.toList());
+  final ClickHandler _clickHandler = ClickHandler();
   Game? _game;
 
   GameState _preGameState = GameState();
   GameState? _gameState;
+  TileLocation? _source;
```

Comment:
Maybe we could wrap this in a built value called `InteractionState`?

Reply:
Renamed to _selectedLocation